### ISSUE TITLE
Document how the app works for contributors

### ISSUE THREAD

**Issue by howdoicomputer** [sfbrigade/adopt-a-drain#109]
Documenting application components like the `load_drains` rake task and how it is used to populate the database. At least, I think that's what it does. This would help contributors quickly onboard themselves and be able to contribute as well as help other brigades fork this application for their own changes.

I don't mind doing this.

**Comment by jszwedko** [comment]
@howdoicomputer good call, I also realized this tonight and had pushed https://github.com/sfbrigade/adopt-a-drain/commit/482a54b331c062d402d7a235e82d9a6190bde047 . The docs could probably use a little more polishing around development though. Let me know what you think.

**Comment by howdoicomputer** [comment]
I was thinking that I would improve the wiki with architectural information on how the application functions.

So the [localizing the application wiki article](https://github.com/sfbrigade/adopt-a-drain/wiki/Localizing-the-adopt-a-platform-to-your-community) is very useful and was thinking about writing information that is moving down a similar line but more focused on how the code works internally so a potential contributor can grok the codebase more quickly.

I'd probably start from the perspective of models.

User Model ->
      * How does authentication work
      * How does session handling work

Thing Model ->
      * What is a 'thing'

Reminder Model ->
      * How does reminders work

Then from a front-end perspective I can go over `main.js.erb` and document that as well since it's almost 800 lines of JavaScript code that does... stuff.

Let me know if this is welcome and I can get started on going through the codebase.

**Comment by jszwedko** [comment]
@howdoicomputer I think that would be very useful work to make it easier for new developers to quickly ramp up and contribute. Thanks for starting this discussion!
